### PR TITLE
Removed game from Standard Projects

### PR DESCRIPTION
**Pull request by sienmich**

### CODE REVIEW

File: src/Player.ts [hunk 1]
```diff
@@ -1682,7 +1682,7 @@ export class Player implements ISerializable<SerializedPlayer> {
     // TODO: Make standard projects static for the game.
```
Comment (sienmich) [inline src/Player.ts:1682]:
```suggestion
```

File: src/Player.ts [hunk 2]
```diff
@@ -1679,10 +1679,9 @@ export class Player implements ISerializable<SerializedPlayer> {
   // Public for testing
   public getPlayableStandardProjects(): Array<StandardProjectCard> {
     // TODO(kberg): Filter playability based on the project's reserve units.
-    // TODO: Make standard projects static for the game.
     return new CardLoader(this.game.gameOptions)
       .getStandardProjects().sort((a, b) => a.cost - b.cost)
-      .filter((card) => card.canAct(this, this.game))
+      .filter((card) => card.canAct(this))
```
Comment (Lynesth) [inline src/Player.ts:1684]:
```ts
.getStandardProjects()
      .filter((card) => card.name !== CardName.SELL_PATENTS_STANDARD_PROJECT && card.canAct(this))
      .sort((a, b) => a.cost - b.cost);
```

**Comment by kberg** [comment]
My hero.

Looks great, but the tests need attention.

File: src/Player.ts (hunk 2 above)
Comment (sienmich) [reply src/Player.ts]: Sure, changed

**Comment by sienmich** [comment]
@kberg, thanks, tests fixes

**Review by kberg — APPROVED** [review]
Thank you! in it goes.

**Pull request merged**